tests/backends/mysql/test_creation.py

```diff
@@ -56,7 +56,18 @@ def test_clone_test_db_database_exists(self):
                 creation._clone_test_db("suffix", verbosity=0, keepdb=True)
                 _clone_db.assert_not_called()
 
-    def test_clone_test_db_options_ordering(self):
+    @mock.patch("subprocess.Popen")
+    def test_clone_test_db_unexpected_error(self, mocked_popen):
```

Comment:
```suggestion
    @mock.patch("sys.stdout", new_callable=StringIO)
    @mock.patch("sys.stderr", new_callable=StringIO)
    @mock.patch("subprocess.Popen")
    def test_clone_test_db_unexpected_error(self, mocked_popen, *mocked_objects):
```
We should capture the logs similar to the other tests